src/EventJournal.md

```diff
@@ -0,0 +1,75 @@
+## Event journal
+
+The event journal is a distributed data structure that stores the history of mutation actions on data structures such as map or cache. Each action on the map or cache which modifies its contents (such as `put`, `remove` or scheduled tasks which are not triggered by using the public API) will create an event which will be stored in the event journal. The event will store the event type as well as the key, old value and updated value for the entry (when applicable). As a user, you can only append to the journal indirectly by using the map and cache methods or by configuring expiration and eviction. 
```

Comment:
Also we're saying this Event Journal is used only by JET.
Why and how is it used for JET?
Is the EventJournal being used on anything else right now?
What would happen if I created an EventJournal on a MAP in my IMDG but I never used JET?